src/components/Navbar/MenuItems.tsx

```diff
@@ -74,27 +74,83 @@ export function TopLeftNavItems({
     </Dropdown>
   ) : null
 
+  const menuItems = !mobile
+    ? [
+        {
+          key: 'index',
+          label: (
+            <Link href="/">
+              <a style={{ display: 'inline-block' }}>{<Logo />}</a>
+            </Link>
+          ),
+        },
+        {
+          key: 'projects',
+          label: (
+            <Link href="/projects">
+              <a {...menuItemProps}>{t`Projects`}</a>
+            </Link>
+          ),
+        },
+        {
+          key: 'docs',
+          label: (
+            <Link href="https://info.juicebox.money/">
+              <a {...externalMenuLinkProps}>{t`Docs`}</a>
+            </Link>
+          ),
+        },
+        {
+          key: 'blog',
+          label: (
+            <Link href="https://info.juicebox.money/blog">
+              <a {...externalMenuLinkProps}>{t`Blog`}</a>
+            </Link>
+          ),
+        },
+        {
+          key: 'resources',
+          label: desktopDropDown,
+        },
+      ]
+    : [
+        {
+          key: 'projects',
+          label: (
+            <Link href="/projects">
+              <a {...menuItemProps}>{t`Projects`}</a>
+            </Link>
+          ),
+        },
+        {
+          key: 'docs',
+          label: (
+            <Link href="https://info.juicebox.money/">
+              <a {...externalMenuLinkProps}>{t`Docs`}</a>
+            </Link>
```

Comment:
**question (non-blocking)** wondering if we can just use ExternalLink here instead?

Reply:
The thought was it's a little easier to parse. Its a bit more explicit, and you know we're using a `next/link`. Honestly not married to it, just thought it could simplify things.

Reply:
Hmm, IMO ExternalLink is a simpler (and safer) approach here, because it means the developer doesn't have to remember to add externalMenuLinkProps every time.

I don't feel too strongly though. Approving.